content/docs/0.8.x/manage/shipyard/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Customize a Shipyard
+description: Adapt the Shipyard depending on your processes.
+weight: 25
+keywords: [0.8.x-manage]
+aliases:
+---
+
+After creating a project, you can change the shipyard as explained here. 
+
+
+The most convenient way to change your shipyard is by directly adapting it in the Git *upstream* repository. 
+
+* If you do not have a Git upstream set for your project, please set one since it is the recommended way of using Keptn: [configure Git upstream](../../manage/git_upstream/#create-keptn-project-or-set-git-upstream).
+
+* If you do not have a Git repository to set an *upstream*, you can update the shipyard using: 
+
+  ```
+  keptn add-resource --project=PROJECT --resource=./shipyard.yaml --resourceUri=./shipyard.yaml
+  ```
+
+:warning: Do *not* modify your shipyard in ways that are *not* mentioned in this section. 
```

Comment:
Double negation. Consider changing to "Only modify your shipyard in ways mentioned in this section."

Reply:
I should have noticed that with the double ** not **. 😄